test/functionalTest/cases/1316_create_subscription/post_subs_errors.test

```diff
@@ -225,7 +225,7 @@ payload='
                "q": "temperature>40",
                "geometry": "point",
                "coords": "-40.4,-3.5",
-               "georel": "near;minDistance=5000"
+               "georel": "near;minDistance:5000"
```

Comment:
They are old.

Until now, expression.(georel|coords|geometry) was just blindly stored/retrieved in/from DB without actually parsing them. Thus, when we change `=` to `:` we didn't notice that some .test need an adjustment.

NTC